### ISSUE TITLE
New translation: Estonian

### ISSUE THREAD

**Issue by Eskumu** [squidfunk/mkdocs-material#1285]
Hey,

Here are the translations for Estonian.

I don't know if for `search.language` I should use the best match from existing lunr.js ("en") languages, or Estonian ("et").

``` jinja
{% macro t(key) %}{{ {
  "language": "et",
  "clipboard.copy": "Kopeeri lõikelauale",
  "clipboard.copied": "Kopeeritud",
  "edit.link.title": "Muuda seda lehte",
  "footer.previous": "Eelmine",
  "footer.next": "Järgmine",
  "meta.comments": "Kommentaarid",
  "meta.source": "Lähtekood",
  "search.language": "et",
  "search.placeholder": "Otsi",
  "search.result.placeholder": "Otsimiseks kirjuta siia",
  "search.result.none": "Otsingule ei leitud ühtegi vastet",
  "search.result.one": "Leiti üks tulemus",
  "search.result.other": "Leiti # tulemust",
  "search.tokenizer": "[\s\-]+",
  "skip.link.title": "Keri sisuni",
  "source.link.title": "Ava repositooriumis",
  "toc.title": "Sisukord"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks for the translation! There's not stemmer for Estonian, but (to me) Finnish (which is available) may be pretty close, so either we use Finnish or don't use a stemmer at all.

**Comment by Eskumu** [comment]
Finnish is simmilar to Estonian, but I don't think Finnish stemmer would work, because stemming is quite different in Estonian. So no stemmer is better option.

I can make the canges in code myself, if you're not already working on it?

**Comment by squidfunk** [comment]
Released as part of 4.4.3
